packages/utilities/fast-react-wrapper/src/index.ts

```diff
@@ -199,6 +199,7 @@ export function provideReactWrapper(React: any, designSystem?: DesignSystem) {
             registrations = [];
         },
     };
+    const wrappersCache = new Map();
```

Comment:
I think this needs to be moved up out of the provider function so that these wrappers can be shared across calls to the provider. Basically, if the same type is passed, we want to get back the same React component, even if it's across provider calls.

Reply:
Rightttt I didn't push my reflexion all the way to multiple providers, that makes sense!